src/xdocs/config_coding.xml

```diff
@@ -1221,6 +1221,32 @@ case 5:
          }
          </source>
          </p>
+         <p>
+         To configure the check so that it checks local variables and parameters and it also validates enhanced For Loop Variable is:
+         </p>
+         <source>
+ &lt;module name="FinalLocalVariable"&gt;
+     &lt;property name="tokens" value="VARIABLE_DEF,PARAMETER_DEF"/&gt;
+     &lt;property name="validateEnhancedForLoopVariable" value="true"/&gt;
+ &lt;/module&gt;
+         </source>
+         <p>Example:</p>
+         <p>
+         <source>
+         public class myclass {
+            static int foo( int x, int y) {
+            return x+y;
+         }
+                public static void main (String []args) {
```

Comment:
please fix formatting.
please generate web site by `mvn clean site -Pno-validations`, please make a screenshot of part that you introduced and share it on github.
Verify yourself that it looks pretty.

Reply:
![screenshot from 2018-01-14 00-02-58](https://user-images.githubusercontent.com/32580268/34908977-3b6f6d38-f891-11e7-939c-9e942a76a8e8.jpg)
Done formatting and uploaded the required screenshot, please review the changes so that i will commit accordingly later on.